test/test_sparse.py

```diff
@@ -561,7 +557,7 @@ def test_tensor(x, res):
             self.assertEqual(res, self.safeToDense(x))
 
             def fn(x):
-                return x.to_dense()
+                return x.to_dense(masked_grad=gradcheck.masked)
             x.requires_grad_(True)
             gradcheck(fn, (x,), check_sparse_nnz=True)
```

Comment:
This tests both `masked=True` and `masked=False`, right?

Reply:
Yes, see the `gradcheck_semantics` decorator.